crates/compiler/gen_llvm/src/llvm/externs.rs

```diff
@@ -160,9 +160,12 @@ pub fn add_default_roc_externs(env: &Env<'_, '_, '_>) {
             }
         }
 
-        unreachable_function(env, "roc_getppid");
-        unreachable_function(env, "roc_mmap");
-        unreachable_function(env, "roc_shm_open");
+        #[cfg(not(target_os = "windows"))]
```

Comment:
that is still not right. you need `env.target_info` and get the operating sytem info from there. target_os is the OS that rust targets to generate the roc binary itself. Here we want to know the target that roc targets and make decisions based on that